util/types/overflow.go

```diff
@@ -18,6 +18,15 @@ import (
 
 	"fmt"
 	"github.com/juju/errors"
+	"github.com/pingcap/tidb/terror"
+)
+
+const (
+	CodeUintAdditionOperationOverflow terror.ErrCode = 1
```

Comment:
It is better to move this definition into util/types/errors.go. And if it is a mysql standard error, please also add the error code mapping in init(). Protocol layer will use this mapping to convert it to corresponding mysql error code.